quiz/QuizData.scala

```diff
@@ -413,6 +413,6 @@ object QuizData {  // to generate tables for a concept connection quizes in late
       "" -> ""
     ).filter(_._1.trim.nonEmpty),
 
-    "" -> Vector()
-  ).filter(_._1.trim.nonEmpty)
+    QuizID("") -> Vector()
+  ).filter(_._1.name.trim.nonEmpty)
```

Comment:
Just curious: Why does each map/vector contain an empty entry that gets filtered out?

Reply:
Reason: I'm longing for the scala version that will include trailing commas:
http://docs.scala-lang.org/sips/completed/trailing-commas.html
But with my trailing empty hack I can copy paste stuff without worring about trailing commas...